test/integration/voldemort/performance/benchmark/Measurement.java

```diff
@@ -92,6 +94,16 @@ public synchronized void recordReturnCode(int code) {
         returnCodes.get(Icode)[0]++;
     }
 
+    public synchronized void recordWarningCode(int code) {
+        Integer Icode = code;

```

Comment:
Nipick: can we follow Java conventions of beginning variable names with a lower case? We also don't typically prefix variables with type information, though I guess that's not a big deal if you want to leave it as is.